docs/en/navigation.yml

```
@@ -571,10 +571,8 @@ nav:
                   - Installation:
                       - Preparation: storage/hwameistor/install/prereq.md
                       - Resource Requirements: storage/hwameistor/install/proresource.md
-                      - Install with Operator: storage/hwameistor/install/deploy-operator.md
-                      - Install with hwameistor-operator: storage/hwameistor/install/deploy-helm.md
-                      - Install DRDB: storage/hwameistor/install/drbdinstall.md
-                      - Install with UI: storage/hwameistor/install/deploy-ui.md
+                      - Hwameistor Installation: storage/hwameistor/install/deploy-ui.md
+                      - Install via HwameiStor-Operator: storage/hwameistor/install/deploy-operator.md
```

```suggestion
                      - Install HwameiStor on UI: storage/hwameistor/install/deploy-ui.md
                      - Install via hwameistor-operator: storage/hwameistor/install/deploy-operator.md
```
- HwameiStor 写法参阅 https://hwameistor.io/
- hwameistor-operator 是一个软件的名称
- 中文也要对应修改